src/components/dev-hub/blog-tag-list.js

```diff
@@ -69,11 +59,13 @@ const BlogTagList = ({ tags = [] }) => {
     return (
         <TagList>
             {isExpanded &&
-                tags.map(t => (
-                    <BlogTag key={t.label} to={t.to}>
-                        {t.label}
-                    </BlogTag>
-                ))}
+                tags
+                    .map(t => (
+                        <BlogTag key={t.label} to={t.to}>
+                            {t.label}
+                        </BlogTag>
+                    ))
+                    .slice(0, MAX_TAG_LIST_SIZE)}
```

Comment:
Should we slice first to reduce the number of items mapped?

Reply:
Great point!